enkit/outputs/commands.go

```diff
@@ -94,11 +95,12 @@ func NewMount(root *Root) *Mount {
 		},
 		root: root,
 	}
-	command.Flags().StringVar(&command.BuildBuddyApiKey, "api-key", "", "build buddy api key used to bypass oauth2")
-	command.Flags().StringVar(&command.BuildBuddyUrl, "url", "", "build buddy url instance")
-	command.Flags().StringVar(&command.ClusterHost, "buildbarn_host", "", "host:port of BuildBarn instance")
+	command.Flags().StringVar(&command.BuildBuddyApiKey, "api_key", "", "build buddy api key used to bypass oauth2")
```

Comment:
as per other PR: most of enkit uses `-` for flag separation. I would stick to the convention used throughout, it is odd to have a single command behave differently.

Reply:
Ack; filed INFRA-550